active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -523,6 +547,28 @@ In a UWP app, this is equivalent to LaunchActivatedEventArgs.Arguments.
 
 # API Details
 
+?? Need to cover resources:
```

Comment:
You could use ` ```XAML ` to add context colour to this code.

Also when you say Need to cover - does that mean you will still be able to set these values in Xaml for both UWP and Win32 **Window**s that are created in WinUI 3?

Reply:
I'm removing this section, we're not trying yet to make this part of the API.